src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -140,6 +142,36 @@ public class DataFileServiceBean implements java.io.Serializable {
      */
     public static final String MIME_TYPE_PACKAGE_FILE = "application/vnd.dataverse.file-package";
     
+    public class UserStorageQuota {
```

Comment:
should we move this out to be its own class? (i.e. any reason for it to be an internal class here?)